### PR TITLE
Label Loading Fix

### PR DESCRIPTION
**Pull request by NeelamMahapatro**
- [ ] Does this PR have a descriptive title that could go in our release notes?
- [ ] Does this PR add any new dependencies?
- [ ] Does this PR modify any existing APIs?
   - [ ] Is the change to the API backwards compatible?
- [ ] Should this result in any changes to our documentation, either updating existing docs or adding new ones?

### CODE REVIEW

File: src/pq_flash_index.cpp [hunk 1]
```diff
@@ -1766,6 +1766,20 @@ template <typename T, typename LabelT> char *PQFlashIndex<T, LabelT>::getHeaderB
...
+    {
+        if (str[start] == ch)
+        {
+            return start;
+        }
+    }
+
+    return std::string::npos;
+}
+
```
Comment (yiylin) [inline src/pq_flash_index.cpp:1782]: image

Comment (yiylin) [reply src/pq_flash_index.cpp:1782]: so strange to have this error

File: src/pq_flash_index.cpp [hunk 2]
```diff
@@ -810,7 +810,7 @@ int PQFlashIndex<T, LabelT>::load_from_separate_paths(uint32_t num_threads, cons
         }
 #endif
         parse_label_file(infile, num_pts_in_label_file);
-        assert(num_pts_in_label_file == this->_num_points);
+        assert(num_pts_in_label_file == this->num_points);
```
Comment (yiylin) [inline src/pq_flash_index.cpp:813]: why there are two "num_points"? can we make them to be one?

**Review by yiylin — APPROVED** [review]

File: src/pq_flash_index.cpp (hunk 1 above)
Comment (NeelamMahapatro) [reply src/pq_flash_index.cpp:1782]: yes, not sure of this. will check later

File: src/pq_flash_index.cpp (hunk 2 above)
Comment (NeelamMahapatro) [reply src/pq_flash_index.cpp:813]: there is one num_points in pqflashindex. but since its assert statement, release build didnot fail I think

**Pull request merged**